autoPlay.user.js

```diff
@@ -408,8 +408,12 @@ function MainLoop() {
 		goToLaneWithBestTarget(level);
 
 		attemptRespawn();
-
-		useAbilities(level);
+		
+		if(isNearEndGame()) {
```

Comment:
`isNearEndGame` will return true in last hour, but I want it to do that only in last 15 minutes.
Modify `isNearEndGame` to return minutes left until the game ends, and do a check based on that.